server/game/setupcardaction.js

```diff
@@ -18,6 +18,7 @@ class SetupCardAction extends BaseAbility {
 
     meetsRequirements(context) {
         return (
+            context.game.readyToStart &&
```

Comment:
Github needs more reaction emojis, my screwups are not accurately reactable in the current list